README.md

```diff
@@ -78,6 +78,8 @@ the 4th byte in the stream.
 
 Enable or disable etag generation, defaults to true.
 
+This will default to `false` if the [`transform` option is passed](#transform).
```

Comment:
This wording is misleading; if the `transform` option is passed, but as `transform: undefined`, I assume this will not default to `false`, but the wording says it will as long as the option is _passed_ (without regard to what value is passed).